app/helpers/geoblacklight_helper.rb

```diff
@@ -164,6 +164,23 @@ def geoblacklight_basemap
     blacklight_config.basemap_provider || 'positron'
   end
 
+  def display_carto
+    return link_to(
+      content_tag(:span, '', class: 'geoblacklight geoblacklight-carto') +
+      t('geoblacklight.tools.open_carto'),
+      carto_link(@document.carto_reference),
+      target: '_blank'
+    ) if @document.carto_reference.present? && Settings.CARTO_DISPLAY
```

Comment:
I wonder if the default should be to display Carto and ArcGIS? Or put another way, test if CARTO_DISPLAY and ARCGIS_DISPLAY are `false` rather than true. If a user does an upgrade to GBL and does not update the settings.yml file, they will lose the links and may not know why.

Reply:
@eliotjordan thank you for thinking about this and reviewing it. As we further discussed I updated the settings so that hide the arcgis and carto links are optional for the users.